src/balena-utils.ts

```
@@ -146,6 +146,29 @@ export async function getReleaseByTags(
 	};
 }
 
+// https://www.balena.io/docs/reference/sdk/node-sdk/#balena.models.release.get
+export async function getReleaseVersion(releaseId: number): Promise<string> {
+	core.debug(`Getting raw version for release ID ${releaseId}`);
+
+	await balena.auth.loginWithToken(
+		core.getInput('balena_token', { required: true }),
+	);
+
+	return await balena.models.release
+		.get(releaseId, { $select: 'raw_version' })
+		.then((release) => {
+			core.debug(JSON.stringify(release));
+			if (release.raw_version) {
+				return release.raw_version;
+			}
+			return '';
```

I would rather throw an error then return empty string so the action doesn't continue